### ISSUE TITLE
Intellisense won't work

### ISSUE THREAD

**Issue by wdsrocha** [tailwindlabs/tailwindcss-intellisense#170]
Sorry to bother, but I'm clueless why the intellisense won't work.

As required, I've added `tailwindcss": "^1.6.2"` on my `package.json` using yarn and I also have the `tailwind.config.js` file, with the following content:

```js
module.exports = {
  theme: {},
  variants: {},
  plugins: [],
  purge: ["./pages/**/*.tsx", "./components/**/*.tsx"],
};
```

I'm using React with Typescript, so after checking issues #72 and #118 I added the following to my VSCode configuration as well
```json
  "tailwindCSS.includeLanguages": {
    "typescript": "javascript",
    "typescriptreact": "javascript"
  },
```

Besides that, I'm using PostCSS, if that matters. More details can be found at the source: https://github.com/wdsrocha/dojo/tree/tailwind-issue

But even after multiple reloads and tweaks here and there, it just won't work. What am I missing? 🤔

**Comment by magicmark** [comment]
I got it to work using this:

```
  "tailwindCSS.includeLanguages": {
    "typescript": "html",
    "typescriptreact": "html"
  }
```

**Comment by wdsrocha** [comment]
Thanks @magicmark, but it wasn't that.

After checking out that intellisense didn't even worked for https://github.com/tailwindlabs/tailwindcss-playground, I realized that maybe there was something wrong with my environment.

And shamefully, I noticed that my VSCode was very outdated, which was the cause for the tailwindcss-intellisense extension to fail silently. 🤦 

Now it is working just fine, thanks. 🎉
